src/backend/gl/src/queue.rs

```diff
@@ -576,16 +576,14 @@ impl CommandQueue {
                     }
                 }
 
-                if rate != 0 {
-                    if self
-                        .share
-                        .legacy_features
-                        .contains(LegacyFeatures::INSTANCED_ATTRIBUTE_BINDING)
-                    {
-                        gl.vertex_attrib_divisor(location, rate);
-                    } else {
-                        error!("Binding attribute with instanced input rate is not supported");
-                    }
+                if self
+                    .share
+                    .legacy_features
+                    .contains(LegacyFeatures::INSTANCED_ATTRIBUTE_BINDING)
+                {
+                    gl.vertex_attrib_divisor(location, rate);
+                } else {
```

Comment:
This branch should either be removed or requires an additional check for rates > 0

Reply:
Good catch, it should be fixed now.